lib/characters.ts

```diff
@@ -17,6 +17,7 @@ export enum CharacterClass {
 }
 
 export type Character = {
+  id: string;
```

Comment:
imo this should be a `number`.

Reply:
the reason i did this was mostly to avoid having to do a bunch of `.toString()` calls throughout the code when looking up data from the JSON CMS data. since JSON doesn't support `number` as an object key we'd have to `.toString()` the ID when doing lookups which could cause confusion down the road, plus this way if we wanted to move to a `UUID` or something fancy in the future it wouldn't be a hard code change